### ISSUE TITLE
Chore: release version 2.2.0

### ISSUE THREAD

**Issue by alexanderankin** [PyFPDF/fpdf2#34]
This is up: https://pypi.org/project/fpdf2/2.2.0rc1/

I just want to understand how ready it is for 2.2.0.

If there are no other font bugs *introduced* as a result of all these new changes, then that is fine.

Also would like to get back to a release cycle that is regular, makes sense, and maybe even with CI on a protected branch to automatically do `"rc$((++version))"`...

**Comment by Lucas-C** [comment]
I think an automated release process would be nice! Maybe based on git tags, so that we would hust have to perform a GitHub release?

For the v2.2.0, do you plan to do some more manual testing, or do you expect something else?

**Comment by Lucas-C** [comment]
Just a note: I've found some remaining bug related to the usage of "core fonts".

I've started working on a PR to fix them. It relies on https://github.com/alexanderankin/pyfpdf/pull/33.
I haven't found the exact fix yet, but I have added tests.

**Comment by alexanderankin** [comment]
ok ive added protection to the master branch, its tempting to remove force pushing on all branches even though its so damn convenient, but thats unrelated - now at least there wont be any accidental version publishes to pypi (only with an up to date review)

**Comment by alexanderankin** [comment]
added #38

**Comment by Lucas-C** [comment]
New PR about the core fonts: https://github.com/alexanderankin/pyfpdf/pull/39

**Comment by Lucas-C** [comment]
@alexanderankin: Do you want to include your fix for https://github.com/alexanderankin/pyfpdf/issues/15 in the new release?

**Comment by Lucas-C** [comment]
I think the branch protection setting you set had an unexpected side effect :(

Now the PR are all pending on unexisting checks: https://github.com/alexanderankin/pyfpdf/pulls

_cf._ https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157

**Comment by alexanderankin** [comment]
ok i dont have time to figure that out right now but ive removed the secret and taken off the check

**Comment by alexanderankin** [comment]
@Lucas-C you are also added on PyPI, feel free to release this new version when you get everything in order. I wont be able to keep up with this pace this upcoming week. thanks.

**Comment by Lucas-C** [comment]
Ok, thank you for the green light! 🚦 
👍

**Comment by Lucas-C** [comment]
@alexanderankin : I checked and I sadly do not have released rights on the Pypi project:
```
HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/
The user 'LucasC' isn't allowed to upload to project 'fpdf2'. See https://pypi.org/help/#project-name for more information.
```

**Comment by alexanderankin** [comment]
Sorry maybe I forgot I sent it now

**Comment by Lucas-C** [comment]
It's good now, thanks!

**Comment by Lucas-C** [comment]
New released published:
- https://github.com/PyFPDF/fpdf2/releases/tag/2.2.0
- https://pypi.org/project/fpdf2/#history
- https://chezsoi.org/lucas/blog/fpdf2-release-2-2-0.html

**Comment by alexanderankin** [comment]
fingers crossed
